modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro

```diff
@@ -143,6 +143,14 @@ definition {
 			value1 = "${value}");
 	}
 
+	// Workaround created because the LPS-168369
```

Comment:
To keep the same pattern of our poshi code, can you replace it for:
"Workaround for LPS-168369"